8/community/Dockerfile

```diff
@@ -9,50 +9,38 @@ EXPOSE 9000
 
 RUN groupadd -r sonarqube && useradd -r -g sonarqube sonarqube
 
-ARG SONARQUBE_VERSION=8.0
+ARG SONARQUBE_VERSION=8.2
```

Comment:
Nothing fails here, this is an arg, it can be overridden. I want the Dockerfile to be final for this version in this PR and only remove overrides in `.travis.yml` when we upload the new zip.

Reply:
it can but if it is not it fails, not sure about docker best practices but it seems odd to me that by default it fails